README.rst

```diff
@@ -111,11 +115,12 @@ You can execute the SCons directly from this repository. For Linux or UNIX::
 
 Or on Windows::
 
-    C:\scons>python scripts\scons.py [arguments]
+    C:\scons>py scripts\scons.py [arguments]
```

Comment:
you may want to add `-3` after py..?
not sure if you can tell the launcher any version >= 3.6?

Reply:
`py -3` (or indeed, without any version specifier) takes the latest installed, unless you've manually configured a default (in a `py.ini`) or a virtualenv is active (uses that one).